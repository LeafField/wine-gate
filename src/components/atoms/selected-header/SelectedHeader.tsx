import React, { FC } from "react";

type Props = {
  title: string;
};

const SelectedHeader: FC<Props> = ({ title }) => {
  return <h3 className="border-b-2 border-red text-heading2">{title}</h3>;
};

export default SelectedHeader;
