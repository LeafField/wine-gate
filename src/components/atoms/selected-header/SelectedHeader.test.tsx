import { screen, render } from "@testing-library/react";
import SelectedHeader from "./SelectedHeader";

describe("SelectedHeaderの単体テスト", () => {
  test("SelectedHeaderコンポーネントが存在することを確認する", () => {
    const { container } = render(<SelectedHeader title="ワイン検索" />);
    expect(screen.getByText("ワイン検索")).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
