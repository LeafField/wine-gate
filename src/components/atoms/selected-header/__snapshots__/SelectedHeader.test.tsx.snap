// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectedHeaderの単体テスト SelectedHeaderコンポーネントが存在することを確認する 1`] = `
<div>
  <h3
    class="border-b-2 border-red text-heading2"
  >
    ワイン検索
  </h3>
</div>
`;
